version.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     # Use "greater than or equal to" range in modules
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = ">= 1.56.1, < 2.0.0"
+      version = ">= 1.62.0"
```

Comment:
add back the higher range limit: `">= 1.62.0, <2.0.0"`

Reply:
Done